sdk/azidentity/azure_cli_credential.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// Enables authentication to Azure Active Directory using Azure CLI to generated an access token.
+type AzureCLICredential struct {
```

Comment:
Also in all other languages as well as in the cli itself the casing Cli is used vs. CLI. Is there a go specific reason to not follow this convention?

Reply:
Copying the same answer I gave Jon: This is a Golang specific requirement. It's idiomatic to the language to use common casing when using acronyms. In this, we differ from the rest of the languages, since the Gopher community and the linter look for this. The current go linter already complains for common acronyms such as ID, which is why we have tenantID and not tenantId like other languages.